library/src/main/java/io/constructor/data/local/PreferencesHelper.kt

```diff
@@ -30,13 +30,13 @@ constructor(@ApplicationContext context: Context, prefFileName: String = PREF_FI
         get() = preferences.getLong(SESSION_LAST_ACCESS, System.currentTimeMillis())
         set(value) = preferences.edit().putLong(SESSION_LAST_ACCESS, value).apply()
 
-    fun getSessionId(sessionIncrementAction: ((String) -> Unit)? = null): Int {
+    fun getSessionId(sessionIncrementAction: ((String) -> Unit)? = null, forceIncrement: Boolean = false): Int {
```

Comment:
Does this force an increment even if the time threshold has not passed?

Reply:
yes, calling app foreground forces it, thats why i added this param